README.md

```diff
@@ -74,7 +74,7 @@ $ node ./bin/ --help
       -p, --port=<n>         Bind bankai to <n> [default: 1337]
       -o, --optimize         Optimize the page and all assets served by bankai [default: false]
       -b, --browse=<app>     Browse the page served by bankai with <app> [default: false]
-      --html.entry=<uri>     Serve client js at <uri> [default: bundle.js]
+      --html.script=<uri>     Serve client js at <uri> [default: bundle.js]
```

Comment:
Is this really warranted? The breaking change could be avoided easily by transforming `htmlOpts` to fit `create-html` API at [handler-html#L19](https://github.com/yoshuawuyts/bankai/pull/57/commits/3efb2aaac102eed5e21617dd8d9413daa03309fd#diff-e4027ddeefb3dca506057f536fd20752R19)